changelog_unreleased/javascript/16349.md

```diff
@@ -0,0 +1,14 @@
+#### Avoid line breaks in import attributes (#16349 by @fisker)
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+import a60 from "./aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.json" assert { type: "json" };
+
+// Prettier stable
+import a60 from "./aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.json" assert { type:
+  "json" };
+
+// Prettier main
+import a60 from "./aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.json" assert { type: "json" };
+```
```

Comment:
I used `a10`, ... `a80`  in #16340 because there were multiple import statements. Seeing `a60` in isolation can be a bit confusing. Something like this can communicate the change a bit better:

```suggestion
<!-- prettier-ignore -->
```jsx
// Input
import something from "./some-very-very-very-very-very-very-very-very-long-path.json" assert { type: "json" };

// Prettier stable
import something from "./some-very-very-very-very-very-very-very-very-long-path.json" assert { type:
  "json" };

// Prettier main
import something from "./some-very-very-very-very-very-very-very-very-long-path.json" assert { type: "json" };
```